platforms-serverless-vercel/vercel-cli-cache-repro/api/somename.js

```diff
@@ -0,0 +1,49 @@
+// A copy of index.js with a different name
```

Comment:
What is this used/needed for?

Reply:
Not used but just used to see if it changes something in the logs to have more than 1 function